src/xpack/network.c

```diff
@@ -152,6 +153,25 @@ static char *_state_sender_serialize_feature_flags(StateSender *sender) {
     return json_str;
 }
 
+// skip datetime for non-platform
+static bool _serialize_the_given_property(CustomProperty *property) {
+    bool to_val = true;
+    const char *value;
+    if ((value = getenv(ROX_ENV_MODE_KEY))) {
+        if (!str_equals(value, ROX_ENV_MODE_PLATFORM)) {
+            CustomPropertyType *propType = custom_property_get_type(property);
+            if (propType == &ROX_CUSTOM_PROPERTY_TYPE_DATETIME) {
+                // skip DateTime
+                to_val = false;
+            }
+            // free the property post check no matter what
+            custom_property_free(propType);
+        }
+        // return true for platform
+    }
+    return to_val;
+}
+
```

Comment:
👍 

there's also a change in flags serialization (adding the external type the platform prefer, even though it should probably work without)
if you want to do it, or open a follow up ticket...